server/modules/surveyRdb/persistence/dbActions/tableViewQuery.js

```diff
@@ -9,33 +9,39 @@ const Node = require('../../../../../common/record/node')
 const SchemaRdb = require('../../../../../common/surveyRdb/schemaRdb')
 const NodeDefTable = require('../../../../../common/surveyRdb/nodeDefTable')
 
+const SqlUtils = require('../../../../../common/exprParser/helpers/sqlUtils')
```

Comment:
exprParser/helpers have to be consider as internal helper modules. you can export the sqlUtils methods from exprParser/expression